ci/setup_clusters.sh

```diff
@@ -38,10 +39,13 @@ done
 wait $PIDS
 
 mkdir -p ~/.kube
-KUBECONFIG=$(find /tmp/kind -type f | tr \\n ':') kubectl config view --flatten > ~/.kube/config
+KUBECONFIG=$(find /tmp/kind -type f | tr \\n ':') kubectl config view --flatten >> ~/.kube/config
+
+kubectl config view
```

Comment:
Do we need to see the entire kubeconfig here? If not, maybe using `--minify` would be better to reduce noise.